### ISSUE TITLE
Autosizing extra div

### ISSUE THREAD

**Issue by jrocketfingers** [JorrinKievit/react-ts-tradingview-widgets#55]
Autosizing a component cannot fit the height properly due to one extra div being added somewhere within the widget itself.

<details><summary>Image demonstrating the problem</summary>
<p>

![image](https://github.com/JorrinKievit/react-ts-tradingview-widgets/assets/2845244/6fc148a0-8812-412b-a070-d554fc524830)

</p>
</details> 
<details><summary>Relevant snippet of JSX</summary>
<p>

```jsx
<div className="flex flex-row gap-5">
  <div className="flex flex-col"> {/* utility */}
    <FundamentalData symbol={`NASDAQ:${stock}`} autosize={false}></FundamentalData>
  </div>
  <div className="flex flex-col flex-1"> {/* utility */}
    <Timeline symbol={`NASDAQ:${stock}`} autosize={true}></Timeline>
  </div>
</div>
```
</p>
</details> 

**Affected widgets (that I've tested so far):**
- Timeline
- MiniChart
- SymbolOverview

**Version of react-ts-tradingview-widgets:**
v1.2.2

**Comment by JorrinKievit** [comment]
Scratch that. You can add a containerId to the component, and use that id to style that div for now until a better solution is found

**Comment by JorrinKievit** [comment]
Managed to quickly figure it out, should be fixed in 1.2.3

**Comment by jrocketfingers** [comment]
Thanks for a quick solution, happy to hear that! Will be reverting back to your components as soon as it's released.

Thanks for the library as well, it really makes working with tradingview widgets bearable 🙇
